packages/fast-permutator/README.md

```diff
@@ -1,9 +1,9 @@
 # FAST permutator (Beta)
-Creates all possible data permutations from a JSON Schema.
+The FAST permutator creates all possible data permutations from a given JSON Schema, allowing for deep testing or display of complex data sets.
 
 ## Use
 ### Permutating data sets
-The main export of the permutator is to recieve a schema and any schema references.
+The main export of the `permutator` is to recieve a schema and any schema references. For example, take this simple schema:
```

Comment:
don't need backticks here, this is not a terminal command or a piece of usable code, instead if we want to explain the main export, can we show this below with the addition of const permutator = require('@Microsoft/fast-permutator');

Reply:
on second thought can  you add in a preliminary block saying how to inlucde the library with const permutator etc.

Reply:
on second thought can  you add in a preliminary block saying how to include the library with const permutator etc.

Reply:
Sure